packages/ui/src/i18n/dictionaries/authenticator/nl.ts

```diff
@@ -21,17 +23,30 @@ export const nlDict: AuthenticatorDictionary = {
   Email: 'E-mail',
   'Enter your code': 'Vul je code in',
   'Enter your email': 'Vul je e-mail in',
+  'Enter your Password': 'Vul je wachtwoord in',
   'Enter your phone number': 'Vul je telefoonnummer in',
   'Enter your username': 'Vul je gebruikersnaam in',
+  'Enter your Username': 'Vul je gebruikersnaam in',
   'Forgot your password?': 'Wachtwoord vergeten? ',
   'Hide password': 'Verberg wachtwoord',
   'It may take a minute to arrive': 'Het kan even duren voordat deze aankomt',
   Loading: 'Laden',
   'New password': 'Nieuw wachtwoord',
+  'New Password': 'Nieuw Wachtwoord',
   or: 'of',
   Password: 'Wachtwoord',
+  'Password did not conform with policy: Password must have numeric characters':
+    'Wachtwoord voldoet niet aan de vereisten: Wachtwoord moet nummers bevatten',
+  'Password did not conform with policy: Password must have symbol characters':
+    'Wachtwoord voldoet niet aan de vereisten: Wachtwoord moet symbolen bevatten',
+  'Password did not conform with policy: Password must have uppercase characters':
+    'Wachtwoord voldoet niet aan de vereisten: Wachtwoord moet hoofdletters bevatten',
+  'Password did not conform with policy: Password not long enough':
+    'Wachtwoord voldoet niet aan de vereisten: Wachtwoord is niet lang genoeg',
```

Comment:
Similar to the above comment, unfortunately we do not have control over these messages and cannot include the keys in our translations